src/test/java/com/contrastsecurity/ContrastSDKTest.java

```diff
@@ -120,5 +119,20 @@ public void negativeTimeoutsAreNotSetTest() throws IOException {
     	assertNotEquals(connectionTimeout, conn.getConnectTimeout());
     	assertNotEquals(readTimeout, conn.getReadTimeout());
     }
+
+    @Test
+    public void testEnsureApi() throws IOException {

```

Comment:
Good test cleanup. Let's add a few more test cases here: 

* Malformed URL
* Null URL
* Blank URL

But these in separate test methods